engine/admin/logging/awslogs.md

```diff
@@ -97,39 +97,132 @@ $ docker run --log-driver=awslogs \
+```
+
+The following `strftime` codes are supported:
+
+| Code | Meaning                                                          | Example  |
+|:-----|:-----------------------------------------------------------------|:---------|
+| `%a` | Weekday abbreviated name.                                        | Mon      |
+| `%A` | Weekday full name.                                               | Monday   |
+| `%w` | Weekday as a decimal number where 0 is Sunday and 6 is Saturday. | 0        |
+| `%d` | Day of the month as a zero-padded decimal number.                | 08       |
+| `%b` | Month abbreviated name.                                          | Feb      |
+| `%B` | Month full name.                                                 | February |
+| `%m` | Month as a zero-padded decimal number.                           | 02       |
+| `%Y` | Year with century as a decimal number.                           | 2008     |
+| `%y` | Year without century as a zero-padded decimal number.            | 08       |
+| `%H` | Hour (24-hour clock) as a zero-padded decimal number.            | 19       |
+| `%I` | Hour (12-hour clock) as a zero-padded decimal number.            | 07       |
+| `%p` | AM or PM.                                                        | AM       |
+| `%M` | Minute as a zero-padded decimal number.                          | 57       |
+| `%S` | Second as a zero-padded decimal number.                          | 04       |
+| `%L` | Milliseconds as a zero-padded decimal number.                    | 123      |
+| `%f` | Microsecond as a zero-padded decimal number.                     | 000345   |
+| `%z` | UTC offset in the form +HHMM or -HHMM.                           | +1300    |
+| `%Z` | Time zone name.                                                  | PST      |
+| `%j` | Day of the year as a zero-padded decimal number.                 | 363      |
+
 ### aws-multiline-pattern
 
 The `aws-multiline-pattern` option defines a multiline start pattern using a
-regular expression. This option is ignored if `awslogs-datetime-format` is also
-configured.
+regular expression. A log message consists of a line that matches the pattern
+and any following lines that don't match the pattern. Thus the matched line is
+the delimiter between log messages.
 
-For example:
+This option is ignored if `awslogs-datetime-format` is also configured.
+
+> **Note**:
+> Multiline logging performs regular expression parsing and matching of all log
+> messages. This may have a negative impact on logging performance.
+
+For example, to process the following log stream where new log messages start with the pattern `INFO`:
+
+Consider the following log stream, where each log message should start with the
+patther `INFO`:
+
+```none
+INFO A message was logged
+INFO Another multiline message was logged
+     Some random message
+INFO Another message was logged
+```
+
+You can use the strftime expression of `[%b %d, %Y %H:%M:%S]`:
```

Comment:
Looks like this line is misplaced.  Did you mean to have it say "You can use the regular expression of `^INFO`:"?